lib/types/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './commit-message-json';
```

Comment:
```suggestion
export type * from './commit-message-json';
```
We should type export if possible, so compiler knows that it can be removed at runtime.

We need to check the other imports too.